src/components/TopBanner.jsx

```diff
@@ -77,7 +77,7 @@ const TopBanner = ({ bannerImg, pageTitle, pageDesc }) => {
   return (
     <TopScreen>
       <Image
-        alt="Get Involved"
+        alt={bannerAlt ? bannerAlt : ''}
```

Comment:
```suggestion
        alt={bannerAlt ?? ''}
```

Reply:
Do we require an extra prop as `bannerAlt` ? Can't we use `pageTitle` instead as the value for Image alt property ?

Reply:
Good point @ShwetKhatri2001, you're probably right. I'm just going to merge this as-is for now, we can change that in a later PR